tests/easier68k/core/opcodes/test_sub.py

```diff
@@ -128,18 +135,20 @@ def test_sub_disassembles():
     sim.set_register(Register.D0, MemoryValue(OpSize.WORD, unsigned_int=123))
     sim.set_register(Register.D1, MemoryValue(OpSize.WORD, unsigned_int=65535))
 
-    result.execute(sim)
-
-    assert sim.get_register(Register.D1).get_value_unsigned() == 65412  # FF84, or (FFFF - 123)
+    run_opcode_test(sim, result, Register.D1, 0xFF84, 2, [0, 1, 0, 0, 0])
 
-    assert sim.get_program_counter_value() == 2
-
-    # changed by execution
-    assert sim.get_condition_status_code(ConditionStatusCode.C) is False
-    assert sim.get_condition_status_code(ConditionStatusCode.V) is False
-    assert sim.get_condition_status_code(ConditionStatusCode.Z) is False
-    assert sim.get_condition_status_code(ConditionStatusCode.N) is True
-    assert sim.get_condition_status_code(ConditionStatusCode.X) is False
+    # result.execute(sim)
+    #
+    # assert sim.get_register(Register.D1).get_value_unsigned() == 65412  # FF84, or (FFFF - 123)
+    #
+    # assert sim.get_program_counter_value() == 2
+    #
+    # # changed by execution
+    # assert sim.get_condition_status_code(ConditionStatusCode.C) is False
+    # assert sim.get_condition_status_code(ConditionStatusCode.V) is False
+    # assert sim.get_condition_status_code(ConditionStatusCode.Z) is False
+    # assert sim.get_condition_status_code(ConditionStatusCode.N) is True
+    # assert sim.get_condition_status_code(ConditionStatusCode.X) is False
```

Comment:
Commented out lines.
(this may seem repetitive, because it is, but I think that it helps track all the specific changes that should be made)